kmm/camera_positions.py

```diff
@@ -40,3 +42,19 @@ def car_direction(header_path: Path):
         r"CarDirection = \"(.*)\"",
```

Comment:
This is probably originally my code but isn't it kind of scary to read a file with a format (xml) and then just ignore it and parse the raw text looking for some pattern instead?

Maybe parse as xml first and find the right tag and use pattern inside the correct tag's text?

Reply:
Actually after looking at it I think parsing it with regex still makes the most sense, the attribute is just in free text in the xml so we have to find it with regex anyway after first finding which element it is in (the file I'm looking at right now has three elements, `Start`, `Sync` and `End` and `CarDirection` is found inside `Start` which isn't obvious to begin with).

Reply:
It's less code but I think it's better to be stricter and not allow weird stuff to happen

Reply:
It's good to crash when stuff is wrong ^^

Reply:
So the way it would work then is we find all the elements tagged as `Start`, select the first one and then find `CarDirection` with regex in that. But do we know that `CarDirection` is always in an element tagged as `Start`? Seems not obvious?